check-peer-deps.js

```diff
@@ -167,8 +169,33 @@ const checkAllPeerDeps = async () => {
   return Promise.all(promises);
 };
 
+const setOpts = () => {
+  const DEBUG_DEFAULT = false;
+  const NO_INCLUDE_DEV_DEFAULT = false;
+  const MAX_RETRIES_DEFAULT = 2;
+
+  const opts = commandLineArgs([
+    { name: 'debug', alias: 'd', type: Boolean, defaultValue: DEBUG_DEFAULT },
+    { name: 'no-include-dev', type: Boolean, defaultValue: NO_INCLUDE_DEV_DEFAULT },
+    { name: 'max-retries', type: Number, defaultValue: MAX_RETRIES_DEFAULT },
+    { name: 'help', alias: 'h', type: Boolean }
+  ]);
+
+  DEBUG = opts['debug'] ? opts['debug'] : DEBUG_DEFAULT;
+  INCLUDE_DEV = opts['no-include-dev'] ? !opts['no-include-dev'] : !NO_INCLUDE_DEV_DEFAULT;
+  MAX_RETRIES = opts['max-retries'] ? opts['max-retries'] : MAX_RETRIES_DEFAULT;
+
+  return opts;
+};
+
 // Main function
 async function checkPeerDeps() {
+  const opts = setOpts();
+
+  if (opts['help']) {
+    console.log(commandLineUsage(require('./option-descriptions')));
```

Comment:
The program execution actually continues after this, we should immediately exit after printing the help text.